internal/manifest/list.go

```diff
@@ -65,6 +69,32 @@ type ListUpdate struct {
 	MediaType string
 }
 
+type ListOp int
+
+const (
+	ListOpIgnore ListOp = iota + 1
```

Comment:
After `ListOpUpdate` is modified to use `UpdateOldDigest`, there’s no reason for `ListOpIgnore` to exist.

Maybe make the first filled `listOpInvalid` (and then handle it in the `default` case of `switch` statements, just to detect uninitialized fields.